models/db_issue.js

```diff
@@ -0,0 +1,35 @@
+var utils = require('../lib/utils');
+var db = require('../lib/db');
+var log = require('debug')('pulldasher:dbissue');
+
+function DBIssue(issue) {
+   this.data = {
+      number: issue.number,
+      title: issue.title,
+      //pull: issue.associatedPull(),
+   };
+
+   if (isNaN(issue.difficulty())) {
```

Comment:
The result from `issue.difficulty()` can be saved for the `isNaN` test and for `parseInt` so that it doesn't have to run again.